examples/inverse/plot_covariance_whitening_dspm.py

```diff
@@ -118,7 +118,7 @@
     for cov in noise_covs:
         inverse_operator = make_inverse_operator(evokeds[0].info, forward,
                                                  cov, loose=0.2, depth=0.8,
-                                                 rank=274)
+                                                 rank=dict(meg=274))
```

Comment:
any idea why we need to specify it manually? it should not be necessary here I think